checkbox-core-snap/series22/snap/snapcraft.yaml

```
@@ -229,7 +228,11 @@ parts:
       - smartmontools
     override-build: |
       cd src && autoreconf -i && cd -
-      snapcraftctl build
+      export PYTHONPATH=$SNAPCRAFT_STAGE/lib/python3.10/site-packages:$SNAPCRAFT_STAGE/usr/lib/python3/dist-packages
+      for path in $(find "$SNAPCRAFT_STAGE/providers/" -mindepth 1 -maxdepth 1 -type d); do export PROVIDERPATH=$path${PROVIDERPATH:+:$PROVIDERPATH}; done
```

```suggestion
      for path in $(find "$SNAPCRAFT_STAGE/../providers/" -mindepth 1 -maxdepth 1 -type d); do export PROVIDERPATH=$path${PROVIDERPATH:+:$PROVIDERPATH}; done
```

Just for the first provider as the stage area is not yet populated; the PROVIDERPATH env var would then be empty and system providers would conflicts during a local build